scripts/submit_eval_jobs.py

```diff
@@ -120,7 +125,7 @@
             --data_dir /data/bbh \
             --save_dir /output/ \
             --use_vllm \
-            --model /model \
+            --model_name_or_path /model \
```

Comment:
How come using `--model` was fine before? The scripts all use `--model_name_or_path`... Just curious...

Reply:
Yeah, this was a weird one. The error shows up if you specify a `--beaker_subfolder`. If you do this, then the code performs a [re.search](https://github.com/allenai/open-instruct/blob/main/scripts/submit_eval_jobs.py#L262) for `model_name_or_path` so that it can replace this path with a path to the subfolder. If the config uses `model` rather than `model_name_or_path`, this `re.search` doesn't return anything and throws an error; that show I found it.

But I'm still confused actually. For instance, take a look at the command for this [gsm eval job](https://beaker.org/ex/01HQNWZ8CAAYXJ0NJ2PNBDPY5S/tasks/01HQNWZ8CE24V2MH3TV7HEQ1C1/job/01HQNWZJXACM3T6EESHGX2MYBB). There's no `--model_name_or_path` specified. Looking at the script [gsm/run eval.py](https://github.com/allenai/open-instruct/blob/main/eval/gsm/run_eval.py#L256), this should throw an error, but it doesn't seem to.

So I'm actually not sure why it doesn't break as currently written. I can try to debug more but thought I'd check with you first.

Reply:
Yeah, I'm not sure whats going on here rn. @yizhongw might have more insight. I agree the subfolder stuff seems broken as-is, though.

Reply:
OK. Let's wait to merge till we hear back from @yizhongw to make sure I haven't subtly introduced a bug?

Reply:
Just closing the loop on why the pre-pr setup wasn't causing an error: Yizhong pointed out that it's because argparse [allows abbreviations](https://docs.python.org/3/library/argparse.html#allow-abbrev) by default.
So, the parsing code was doing error recovery for us. Still it's probably safer to have the argument names be consistent.

Reply:
Awesome, I didn't know about that feature of argparse! agreed that we should keep argument names consistent.